<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="ProcessToolBtn">
    <property name="geometry">
     <rect>
      <x>2000</x>
      <y>50</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>进程列表</string>
    </property>
   </widget>
   <widget class="QToolButton" name="DumpToolBtn">
    <property name="geometry">
     <rect>
      <x>7000</x>
      <y>50</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>崩溃转储</string>
    </property>
   </widget>
   <widget class="QToolButton" name="VersionToolBtn">
    <property name="geometry">
     <rect>
      <x>12000</x>
      <y>50</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>版本列表</string>
    </property>
   </widget>
   <widget class="QToolButton" name="LogToolBtn">
    <property name="geometry">
     <rect>
      <x>17000</x>
      <y>50</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>日志文件</string>
    </property>
   </widget>
   <widget class="QWidget" name="BackWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="CpuGroupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>381</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>CPU</string>
     </property>
     <widget class="QLabel" name="CpuLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>341</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="GpuGroupBox">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>0</y>
       <width>401</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>GPU</string>
     </property>
     <widget class="QLabel" name="GpuLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>341</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="MemoryGroupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>381</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>Memory</string>
     </property>
     <widget class="QLabel" name="MemoryLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>341</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="DiskGroupBox">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>270</y>
       <width>401</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>Disk</string>
     </property>
     <widget class="QLabel" name="DiskLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>341</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="Menu1">
    <property name="title">
     <string>--</string>
    </property>
    <addaction name="Menu1Action0"/>
    <addaction name="Menu1Action1"/>
   </widget>
   <widget class="QMenu" name="Menu2">
    <property name="title">
     <string>--</string>
    </property>
    <addaction name="Menu2Action0"/>
    <addaction name="Menu2Action1"/>
    <addaction name="Menu2Action2"/>
    <addaction name="Menu2Action3"/>
   </widget>
   <widget class="QMenu" name="Menu4">
    <property name="title">
     <string>--</string>
    </property>
    <addaction name="Menu4Action0"/>
   </widget>
   <addaction name="Menu1"/>
   <addaction name="Menu2"/>
   <addaction name="Menu4"/>
  </widget>
  <widget class="QToolBar" name="ToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="StatusBar"/>
  <action name="Menu2Action0">
   <property name="text">
    <string>--</string>
   </property>
  </action>
  <action name="Menu4Action0">
   <property name="text">
    <string>--</string>
   </property>
  </action>
  <action name="Menu1Action0">
   <property name="text">
    <string>--</string>
   </property>
  </action>
  <action name="Menu2Action1">
   <property name="text">
    <string>--</string>
   </property>
  </action>
  <action name="Menu2Action2">
   <property name="text">
    <string>--</string>
   </property>
  </action>
  <action name="Menu2Action3">
   <property name="text">
    <string>--</string>
   </property>
  </action>
  <action name="Menu1Action1">
   <property name="text">
    <string>--</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
